src/pages/search/index.js

```diff
@@ -0,0 +1,146 @@
+import TextInput from "../../components/form/textInput";
+import { useState, useEffect, useRef } from "react";
+import SearchIcon from "../../assets/icons/searchIcon";
+import "./search.css";
+import BackButton from "../../components/backbutton"
+import { getUsers } from "../../service/apiClient"
+import { Link } from "react-router-dom";
+import CrossBlackIcon from "../../assets/icons/crossBlackIcon";
+import ProfileIcon from "../../assets/icons/profileIcon";
+
+function SearchPage () {
+    // create a state hook that will store the fetched data.
+    const [formData, setFormData] = useState('')
+    const [users, setUsers] = useState([])
+    const [results, setResults] = useState([])
+    const [showMore, setShowMore] = useState(-1)
+    const ref = useRef(null)
+
+    useEffect(() => {
+        getUsers().then(setUsers)
+        
+    }, [])
+   
+    useEffect(() => {
```

Comment:
What's the reasoning behind 
- having two effects making the same HTTP request?
- using two different state hooks to store the same data? (is this because of filtering results later?)

Reply:
Would something like this work?

```js
    useEffect(() => {
        getUsers()
          .then(data => {
            setUsers(data)
            setResults(data)
         })
    }, [])
```

Reply:
ok yea, this seams better